app/components/edit/EditSidebar.jsx

```diff
@@ -87,6 +87,12 @@ const EditSidebar = ({
       allServices[service.id].name = service.name;
     });
   }
+
+  const sortedServices = Object.entries(allServices).sort((a, b) => {
+    if (+a[0] < 0) return 0;
+    if (+b[0] < 0) return 1;
```

Comment:
Once again, this probably won't work well, since casting strings to numbers using the unary `+` operator will not work the way you want it to most of the time. I'd also recommend using lodash's `sortBy` here.